src/db/migrate.rs

```diff
@@ -295,6 +295,20 @@ pub fn migrate(version: Option<Version>, conn: &Connection) -> CratesfyiResult<(
                 DROP FUNCTION normalize_crate_name;
             "
         ),
+        migration!(
+            context,
+            // version
+            11,
+            // description
+            "Allow crate priorities to be stored",
+            // upgrade query
+            "CREATE TABLE crate_priorities (
+                pattern VARCHAR NOT NULL PRIMARY KEY,
```

Comment:
Just in case we decide to allow crate IDs in the future.

```suggestion
                pattern VARCHAR NOT NULL UNIQUE,
```